vme/src/utility.h

```diff
@@ -25,11 +25,21 @@
 class log_buffer
 {
 public:
-    log_buffer(void) { str[0] = 0; }
+    log_buffer() = default;
```

Comment:
This is used by slog and used a char[] array for storing the log - so I first encapsulated the members to see where else it was used and converted it to std::string